lib/lib.go

```diff
@@ -32,16 +45,417 @@ func init() {
 
 // Receivers returns a slice of CoreRequests that defines the full local
 // API of lib methods
-func Receivers(node *p2p.QriNode) []Requests {
+func Receivers(inst Instance) []Requests {
+	node := inst.Node()
+	r := inst.Repo()
```

Comment:
Would prefer not using this temp `r`, and instead just using node.Repo where it's needed.

Reply:
ok, let's debate & cleanup in subsequent PRs